Week1/assets/HYFRepo.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const gitHub = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+
+function createAppendElement(htmlTag, parent) {
+
+    const node = document.createElement(htmlTag);
+    parent.appendChild(node);
+    return node;
+}
+
+function content() {
+
+    const main = document.getElementById('root');
+
+    const header = createAppendElement('header', main);
+    header.setAttribute('class', 'selection-wrapper');
+
+    const label = createAppendElement('label', header);
+    label.setAttribute('class', 'labelcls');
+    label.textContent = 'Select Repository' + ' ';
+
+    const toSelect = createAppendElement('select', header);
+    toSelect.setAttribute('id', 'selections');
+
+    const wrapperDiv = createAppendElement('div', main);
+    wrapperDiv.setAttribute('class', 'main-wrapper');
+
+    const repoInfoDiv = createAppendElement('div', wrapperDiv);
+    repoInfoDiv.setAttribute('id', 'repo-Info');
+
+    const createUlIno = createAppendElement('ul', repoInfoDiv);
+    createUlIno.setAttribute('id', 'repo-info-ul');
+
+    const contInfoDiv = createAppendElement('div', wrapperDiv);
+    contInfoDiv.setAttribute('id', 'cont-info');
+
+    const contSection = createAppendElement('p', contInfoDiv);
+    contSection.setAttribute('class', 'section-name');
+    contSection.textContent = 'Contributions';
+
+    const ulContInfo = createAppendElement('ul', contInfoDiv);
+    ulContInfo.setAttribute('id', 'cont-info-ul');
+
+
+    retrieveData(gitHub, (error, data) => {
+        if (error !== null) {
+            console.error(error.message);
```

Comment:
The assignment says that you should display the errors on the screen instead of printing them to the console. This is because not all users see/know how to access the console output.

Reply:
Thank you very much Hardit. your are right. but the assignment you are seeing now is the updates assignemt, the update done on lasr Sunday after I did and pushed my assignemnt. So my assignment meets the old version assignment.
Anyway, i am trying know in week2 assignment to integrate the updates of the week one.
Regards...